content/docs/gke/authentication.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Troubleshooting guide for authentication and authorization to GCP"
```

Comment:
Although the original issue requested a "troubleshooting" guide, I think this page is more of a full how-to guide. Suggested description:

`Authentication and authorization to GCP`